src/Content/Conversation/Entity/Timeline.php

```diff
@@ -34,6 +34,7 @@
  * @property-read int    $mediaType      Media types that are included in the channel
  * @property-read array  $languages      Channel languages
  * @property-read int    $circle         Circle or timeline this channel is based on
+ * @property-read bool   $publish        Publish the channel
  */
```

Comment:
This list is missing the `valid` property.

Reply:
Done.